build/hivedscheduler/go-build.sh

```diff
@@ -37,6 +37,9 @@ rm -rf ${DIST_DIR}
 mkdir -p ${DIST_DIR}
 
 go build -o ${DIST_DIR}/hivedscheduler cmd/hivedscheduler/*
+if [ ! -z ${1:-} ] && [ $1 == "test" ]; then
+    go test ./pkg/algorithm/
```

Comment:
>./pkg/algorithm/ [](start = 12, length = 16)

Can we also test the whole project, instead of just ./pkg/algorithm/?

Like we build the whole project, go build -o ${DIST_DIR}/hivedscheduler cmd/hivedscheduler/*